samples/Demo/speedometer-consumer/src/main.rs

```diff
@@ -164,6 +170,36 @@ async fn receive_vehicle_speed_updates(
     Ok(sub_handle)
 }
 
+pub fn runningLed(panel: &mut led_driver::ws2811_t, led_color: u32, delay_ms: u64, max_count: i32)
+{
+    let count = if max_count < panel.channel[0].count { max_count } else { panel.channel[0].count };
+    loop {
+        for i in 0..=count-1 {
+            led_driver::setOnlyOneLedToRgb(panel, i as u32, led_color);
+            std::thread::sleep(std::time::Duration::from_millis(delay_ms));
+        }
+        for i in 0..=count-1 {
```

Comment:
make this loop shorter, so that it does not draw the first and last led twice. Now it does: ...5,6,double7,6,5....2,1,double0,1,2.....

Reply:
Fixed